src/_locales/fr.ts

```diff
@@ -1 +1,92 @@
-//Not supported yet, but if you want to help, copy en.ts file contents here and translate, then make a PR
+import { MRT_Localization } from '../MaterialReactTable';
+
+export const MRT_Localization_EN: MRT_Localization = {
```

Comment:
MRT_Localization_FR

Reply:
you can make another commit to update the name if you want

Reply:
Okay